Makefile.template

```diff
@@ -0,0 +1,13 @@
+# For scripted languages it might be unnecessary to change anything except the line below `run:`
```

Comment:
Where? There's also a “not” hidden in the word “**un**necessary”, but I can change the sentence using “not” directly, which would make it a little bit more clear.

What do you think about:

```Makefile
# For scripted languages it might **not** be necessary to change anything except the line below `run:`
```

Or is my understanding of the english language fooling me, and both sentences do actually something very different to what I want to say here? English isn't my primary language, only secondary and nearly exclusively used while reading.